### PR TITLE
update interactive `launch`

### PR DESCRIPTION
**Pull request by hoxhaeris**
Add a few in-between steps when selecting the version, to avoid having to redirect users to the `release-controller` page.

### CODE REVIEW

File: pkg/slack/modals/launch/launch.go [hunk 1]
```diff
@@ -8,15 +8,44 @@ import (
...
 	"sync"
 )
 
-// Identifier is the view identifier for this modal
-const Identifier modals.Identifier = "launch"
-const Identifier2ndStep modals.Identifier = "launch2ndStep"
-const Identifier3rdStep modals.Identifier = "launch3ddStep"
+const (
+	IdentifierInitialView        modals.Identifier = "launch"
+	Identifier3rdStep            modals.Identifier = "launch3ddStep"
```
Comment (bradmwilliams) [inline pkg/slack/modals/launch/launch.go:20]: Should `launch3ddStep` be `launch3rdStep`?

File: pkg/slack/modals/launch/launch.go [hunk 2]
```diff
@@ -8,15 +8,44 @@ import (
...
+	launchArchitecture   = "architecture"
+	launchParameters     = "parameters"
+	launchVersion        = "version"
+	launchStepContext    = "context"
+	defaultPlatform      = "aws"
+	defaultArchitecture  = "amd64"
+	launchMode           = "launch_mode"
+	launchModeVersion    = "version"
+	launchModePR         = "pr"
+	launchModePRKey      = "One or multiple PR-s"
```
Comment (bradmwilliams) [inline pkg/slack/modals/launch/launch.go:45]: `PR-s` seems weird to me.  Can it not `PR's` or just `PR's`?

**Review by bradmwilliams — CHANGES_REQUESTED** [review]

File: pkg/slack/modals/launch/launch.go (hunk 1 above)
Comment (hoxhaeris) [reply pkg/slack/modals/launch/launch.go]: Fixed

File: pkg/slack/modals/launch/launch.go (hunk 2 above)
Comment (hoxhaeris) [reply pkg/slack/modals/launch/launch.go]: Fixed

**Review by bradmwilliams — APPROVED** [review]

**Pull request merged**